config/etcd-io/sig-etcd/teams.yaml

```diff
@@ -113,3 +113,19 @@ teams:
           gofail: triage
           raft: triage
           website: triage
+  release-etcd:
+    description: Granted permission to release etcd-io/etcd
+    members:
+      - ahrtr
+      - fuweid
+      - ivanvc
+      - jmhbnz
+      - serathius
+      - siyuanfoundation
+      - spzala
+      - wenjiaswe
+    privacy: closed
+    repos:
+      # Permission set to triage during bau activities
+      # During release windows this will be bumped to `maintain`
+      etcd: triage
```

Comment:
We can easily bump this when a release is planned as etcd maintainers are `OWNERS` to `/lgtm /approve` a pr from a reviewer to bump it for the release then it can be reverted to bau afterward.

This ensures principle of least privileges is maintained.